### ISSUE TITLE
organization

### ISSUE THREAD

**Issue by wrightaq** [HRUchiha/front-end-capstone#16]
I don't know if this is best practice or not but I feel like I've seen or heard that it's cleaner to not have too much code in the render method for components and rather use references to what should be rendered. But honestly I don't know if that's a real best practice thing. I think your code looks really clean , but I can see how using more references to code in the render can make it easier to visualize what will be put on the page and how it will generally be organized. An example of a lot of code in the render is the Question component.

**Comment by Y-nick** [comment]
Good point, thank you! I'll try to re-factor some of the code out of the render
